Makefile

```diff
@@ -46,7 +46,7 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:generateEmbeddedObjectMeta=true webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:generateEmbeddedObjectMeta=true,allowDangerousTypes=true,ignoreUnexportedFields=true webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
`ignoreUnexportedFields=true` is kind of dangerous, but I don't see another way? maybe we should at least check in CI/before release what this looks like manually/find a different override (perhaps a kubebuilder tag?)